io.openems.common/src/io/openems/common/event/EventReader.java

```diff
@@ -116,4 +117,31 @@ public ZonedDateTime getZonedDateTime(String propertyId, String timeZone) {
 		return this.getZonedDateTime(propertyId, ZoneId.of(timeZone));
 	}
 
+	@Override
+	public String toString() {
+		StringBuilder asString = new StringBuilder("EventReader{Event={Topic=");
```

Comment:
Commonly we use squared brackets in toString methods in OpenEMS. This is the way Eclipse IDE autogenerates toString methods. Please consider changing in a future PR.